pkg/config/config_test.go

```diff
@@ -384,8 +384,8 @@ func TestNewCreateChannelTxFailure(t *testing.T) {
 				profile.Policies[ReadersPolicyKey].Rule = "ANYY Readers"
 				return profile
 			},
-			err: errors.New("failed to create default config template: failed to create new channel group: " +
-				"failed to add policies: invalid implicit meta policy rule: 'ANYY Readers' error: unknown rule type " +
+			err: errors.New("creating default config template: failed to add policies to channel group: " +
+				"invalid implicit meta policy rule: 'ANYY Readers': unknown rule type " +
 				"'ANYY', expected ALL, ANY, or MAJORITY"),
```

Comment:
seems very verbose but not worth changing right now.

Reply:
It's been trimmed down some. Is it reasonable for the consumer to know which rules are valid?